runtime/oti/j9nonbuilder.h

```diff
@@ -2054,16 +2054,12 @@ typedef struct J9BCTranslationData {
 #define BCT_Java7MajorVersionShifted  0x33000000
 #define BCT_Java8MajorVersionShifted  0x34000000
 #define BCT_Java9MajorVersionShifted  0x35000000
-#define BCT_Java10MajorVersionShifted 0x36000000
```

Comment:
Removing old versions should be done in a separate PR (and would then also remove the macros for 2, 5, 6, 7 and 9 as well).

Reply:
> remove the macros for 2, 5, 6, 7 and 9 as well

Those versions have more references, and the cleanup is not straightforward. I think those area need to be examined more carefully which I would prefer in another PR after the easy cleanup PR.

Reply:
I think that cleanup should remove `J2SE_VERSION` altogether in favour of using `JAVA_SPEC_VERSION`. There's no longer a reasonable use-case where those tests would disagree.

Reply:
That will be an even larger refactoring, I attempted couple of years ago at https://github.com/eclipse-openj9/openj9/pull/5370.
Would suggest do it in another separated PR.